public/js/index.js

```diff
@@ -25,13 +25,6 @@ slider.addEventListener("change", ({ target: { value } }) => {
 window.addEventListener("load", async () => {
   config = await fetch("/config").then((r) => r.json());
```

Comment:
We can remove this fetch for the cofing and instead use template variables from ejs.